api/default_networks.go

```diff
@@ -31,9 +31,9 @@ func mainnet(stageName string) params.Network {
 		ChainName:              "Mainnet",
 		DefaultRPCURL:          fmt.Sprintf("https://%s.api.status.im/nodefleet/ethereum/mainnet/", stageName),
 		DefaultFallbackURL:     fmt.Sprintf("https://%s.api.status.im/infura/ethereum/mainnet/", stageName),
-		DefaultFallbackURL2:    fmt.Sprintf("https://%s.api.status.im/grove/ethereum/mainnet/", stageName),
-		RPCURL:                 "https://eth-archival.rpc.grove.city/v1/",
-		FallbackURL:            "https://mainnet.infura.io/v3/",
+		DefaultFallbackURL2:    "https://mainnet.infura.io/v3/",
```

Comment:
```
	if proxyProvider.Enabled {
		key := ProviderStatusProxy
		keyFallback := ProviderStatusProxy + "-fallback"
		keyFallback2 := ProviderStatusProxy + "-fallback2"
		urls[key] = network.DefaultRPCURL
		urls[keyFallback] = network.DefaultFallbackURL
		urls[keyFallback2] = network.DefaultFallbackURL2
		keys = []string{key, keyFallback, keyFallback2}
		authMap[key] = proxyProvider.User + ":" + proxyProvider.Password
		authMap[keyFallback] = authMap[key]
		authMap[keyFallback2] = authMap[key]
	}
	keys = append(keys, []string{"main", "fallback"}...)
	urls["main"] = network.RPCURL
	urls["fallback"] = network.FallbackURL
```
We do not use authentication for RPCURL. If I understand correctly it should be:
```
		DefaultRPCURL:          fmt.Sprintf("https://%s.api.status.im/nodefleet/ethereum/mainnet/", stageName),
		DefaultFallbackURL:     fmt.Sprintf("https://%s.api.status.im/infura/ethereum/mainnet/", stageName),
		DefaultFallbackURL2:    fmt.Sprintf("https://%s.api.status.im/grove/ethereum/mainnet/", stageName),
		RPCURL:                 "https://mainnet.infura.io/v3/",
		FallbackURL:            "https://eth-archival.rpc.grove.city/v1/",
		BlockExplorerURL:       "https://etherscan.io/",
```
```
StatusProxy - Node Fleet
StatusProxy - Infura
Direct Infura
StatusProxy - Grove
Direct Grove
```
and if we need to have `proxy-grove` lower than `direct-infura`, then we should probably add "priority" field to params.network, and respect it in `rpc/client.go` `getEthClients` method.

Reply:
@friofry good point, I missed that usage. Thanks.
Need to think a bit about how to do it painless for now.

Reply:
Now should be good, providers priority is determined by the `getProviderPriorityByURL` function.